presto-main/src/main/java/com/facebook/presto/FullConnectorSession.java

```diff
@@ -46,6 +47,7 @@
     private final boolean isLegacyTimestamp;
```

Comment:
Now you can remove `isLegacyTimestamp` (it's `== SystemSessionProperties.isLegacyTimestamp(session)`)

Reply:
We can get rid of most of these now. I was trying to minimize changes, but might as well remove the duplication.

Reply:
I'll keep it as a field so it's free to access (looking up session properties has a non-zero cost which could have an impact if, say, a connector called this method for every row in a query).